dotcom-rendering/src/components/FrontSectionTitle.tsx

```diff
@@ -113,17 +159,58 @@ export const FrontSectionTitle = ({ title, collectionBranding }: Props) => {
 				isContainerBranding,
 				isFrontBranding,
 			} = collectionBranding;
+			const isAdvertisingPartnerOrExclusive =
+				logo.label.toLowerCase() === 'advertising partner' ||
+				logo.label.toLowerCase() === 'exclusive advertising partner';
 			if (isFrontBranding || isContainerBranding) {
 				return (
 					<div css={titleStyle}>
 						{title}
-						<>
-							<p css={labelStyles}>{logo.label}</p>
-							<Badge imageSrc={logo.src} href={logo.link} />
-							<a href={aboutThisLink} css={aboutThisLinkStyles}>
+						{isAdvertisingPartnerOrExclusive &&
+						inAdvertisingPartnerABTest ? (
+							<div css={advertisingPartnerDottedBorder}></div>
+						) : (
+							<div></div>
```

Comment:
Would it be more semantic to use a horizontal rule here, and nothing if we do not need it?

```suggestion
							<hr css={advertisingPartnerDottedBorder} />
						) : (
							null
```

Alternatively, you could add these styles conditionally to the following div?

Reply:
When you said in a previous comment to have an `hr` I didn't know that I could style it and thank you for the tip that anything could be styled. Done in 7656f983c447ba3f271117b5c14d38ef7fafa3b7